ibc-core/ics04-channel/src/handler/recv_packet.rs

```diff
@@ -252,16 +252,14 @@ where
             let packet_rec = ctx_b.get_packet_receipt(&receipt_path_on_b);
             match packet_rec {
                 Ok(_receipt) => {}
-                Err(ContextError::PacketError(PacketError::MissingPacketReceipt(sequence)))
-                    if sequence == msg.packet.seq_on_a => {}
-                Err(e) => return Err(e),
```

Comment:
This `match` arm needed to be removed as the `ctx_b.get_packet_receipt` method no longer returns a `ContextError`/`HandlerError`: it now returns a `HostError` which, at least with how it is defined as of this writing, cannot encode a packet sequence.

Reply:
Removing this match arm ended up breaking the [recv_packet_happy_path](https://github.com/cosmos/ibc-rs/blob/0202decd8e6eab5c0e2ba3cdc9ec615a40cad488/tests-integration/tests/core/ics04_channel/recv_packet.rs#L107) test. The test needed to receive the packet receipt context that it was now missing, but once that was added, the test went back to passing.